src/lexer.coffee

```diff
@@ -849,6 +856,8 @@ MULTI_DENT = /^(?:\n[^\n\S]*)+/
 
 JSTOKEN    = /^`[^\\`]*(?:\\.[^\\`]*)*`/
 
+IMPORT     = /^import .+/
+
```

Comment:
This is not the way to go. Have a look at how other keywords, such as `if`, are tokenized.